tachyon/math/circle/generator/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("//bazel:tachyon_cc.bzl", "tachyon_cc_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+tachyon_cc_binary(
+    name = "generator",
+    srcs = ["generator.cc"],
+    data = ["cpu.h.tpl"],
+    deps = [
+        "//tachyon/base/console",
```

Comment:
`#include "absl/strings/str_replace.h"`, `#include "tachyon/base/strings/string_number_conversions.h"`, and `#include "tachyon/base/containers/container_util.h"`
 not listed as dep? (from generator.cc deps)

Reply:
Hmm, I think it's better to put those explicitly. Thanks!

Reply:
`container_util` still hasn't been added 🤔, but it's trivial I suppose.

Reply:
Nice catch! I think it's rather to be deleted. It's not being used.